### ISSUE TITLE
Flaky test: applyPermissions_wrongUrlFails

### ISSUE THREAD

**Issue by rorbech** [realm/realm-java#7065]
From https://ci.realm.io/blue/organizations/jenkins/realm%2Frealm-java/detail/PR-7053/2/pipeline
```
io.realm.PathLevelPermissionsTests > applyPermissions_wrongUrlFails[CIEmulator(AVD) - 10] FAILED 
	java.lang.NullPointerException: Attempt to invoke virtual method 'java.io.File android.content.Context.getFilesDir()' on a null object reference
	at io.realm.BaseRealm.<init>(BaseRealm.java:132)
```

**Comment by rorbech** [comment]
Once again in https://ci.realm.io/blue/organizations/jenkins/realm%2Frealm-java/detail/PR-7068/1/tests

**Comment by edualonso** [comment]
This test is a part of PathLevelPermissionsTests.java, which was removed from the v10 branch, so should not be a concern once v10 is merged into master.
